### PR TITLE
Newsletter-215:Translate into Japanese

### PR DESCRIPTION
**Pull request by azuchi**
This translation contains fix #846. Please point out if this fix is not necessary.

### CODE REVIEW

File: _posts/ja/newsletters/2022-08-31-newsletter.md [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+{% include linkers/issues.md v=2 issues="23202,2275,2387,1652,627,718,705,722,24114" %}
+[lnd 0.15.1-beta]: https://github.com/lightningnetwork/lnd/releases/tag/v0.15.1-beta
+[news175 verify]: /ja/newsletters/2021/11/17/#bitcoin-core-22934
+[news87 verify]: /ja/newsletters/2020/03/04/#bips-886
+[news83 verify]: /ja/newsletters/2020/02/05/#schnorr
+[raw interchange]: https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2022-August/020887.html
+[moody slip15]: https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2022-August/020888.html
+[rusnak slip15]: https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2022-August/020892.html
+[raw slip15]: https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2022-August/020893.html
+[aliases]: /en/newsletters/2022/07/13/#lnd-5955
```
Comment (bitschmidty) [inline _posts/ja/newsletters/2022-08-31-newsletter.md:121]:
```suggestion
[aliases]: /ja/newsletters/2022/07/13/#lnd-5955
```

**Review by bitschmidty — COMMENTED** [review]
@azuchi one small change suggested

File: _posts/ja/newsletters/2022-08-31-newsletter.md (hunk 1 above)
Comment (azuchi) [reply _posts/ja/newsletters/2022-08-31-newsletter.md]: @bitschmidty thanks. I fixed.

**Review by bitschmidty — APPROVED** [review]
ACK, thanks @azuchi !

**Pull request merged**